runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/RunecraftAction.java

```diff
@@ -65,7 +66,22 @@ public enum RunecraftAction implements ItemSkillAction
 	NATURE_RUNE(ItemID.NATURE_RUNE, 44, 9, false, true),
 	LAW_RUNE(ItemID.LAW_RUNE, 54, 9.5f, false, true),
 	DEATH_RUNE(ItemID.DEATH_RUNE, 65, 10, false, true),
-	BLOOD_RUNE(ItemID.BLOOD_RUNE, 77, 24.425f, true),
+	ZEAH_BLOOD_RUNE(ItemID.BLOOD_RUNE, 77, 24.425f, true)
+		{
+			@Override
+			public String getName(final ItemManager itemManager)
+			{
+				return "Blood Rune (Zeah)";
+			}
+		},
```

Comment:
These blocks should be dedented one level
```suggestion
	{
		@Override
		public String getName(final ItemManager itemManager)
		{
			return "Blood Rune (Zeah)";
		}
	},
```

Reply:
Done. This breaks autoformatting, but it is consistent with the goldsmithing gauntlets formatting.